src/main.c

```diff
@@ -116,6 +116,7 @@ char *startin_path = NULL;
 uint8_t keymap = 0; // KERNAL's default
 int window_scale = 1;
 float screen_x_scale = 1.0;
+float window_transparency = 1.0;
```

Comment:
If the SDL routine sets opacity, I think that's what we should call the command line switch and the variable.  Transparency being the counterpart to opacity.